src/main/java/org/jenkinsci/maven/plugins/hpi/RunMojo.java

```diff
@@ -724,7 +724,7 @@ protected Artifact getJenkinsWarArtifact() throws MojoExecutionException {
             if (jenkinsWarId!=null)
                 match = (a.getGroupId()+':'+a.getArtifactId()).equals(jenkinsWarId);
             else
-                match = (a.getArtifactId().equals("jenkins-war") || a.getArtifactId().equals("hudson-war")) && a.getType().equals("war");
+                match = (a.getArtifactId().equals("jenkins-war") || a.getArtifactId().equals("hudson-war")) && a.getType().equals("executable-war");
```

Comment:
Yeah, probably could do that, if we wanted to allow the newer plugin to be used in older POMs.